src/vendor/jquery.js

```diff
@@ -1,4 +0,0 @@
-/*! jQuery v3.2.1 | (c) JS Foundation and other contributors | jquery.org/license */
```

Comment:
We don't need jQuery for anything? Modus is built on Bootstrap, which in turn uses jQuery.

Reply:
Oh I see - it's now part of the Modus subfolder. Never mind me.